crates/tabby-common/src/config.rs

```diff
@@ -73,8 +73,9 @@ impl Config {
 
     fn validate_dirs(&self) -> Result<()> {
```

Comment:
there's no longer need to validate dirs as config_index_to_id is always unique?

Reply:
I'm not sure. Would hash value be conflict? Or we can remove this method.

Reply:
the id encode is not a hash, it's an 1-1 mapping allowing bi-directional conversion.